dexs/hydradx/index.ts

```diff
@@ -14,8 +15,42 @@ const fetch = async (timestamp: number) => {
   const response: IAPIResponse = (await fetchURL(URL));
   const dailyVolume = response[0].volume_usd;
 
+  // Fetch fees data from GraphQL endpoint
+  let dailyFees = 0;
+  try {
+    const feesQuery = `
+      query MyQuery {
+        platformTotalVolumesByPeriod(filter: {period: _24H_}) {
+          nodes {
+            omnipoolFeeVolNorm
+            stableswapFeeVolNorm
+            xykpoolFeeVolNorm
+          }
+        }
+      }
+    `;
+
+    const feesResponse = await request("https://galacticcouncil.squids.live/hydration-pools:whale-prod/api/graphql", feesQuery);
```

Comment:
would we be able to pull the fee & volume data for any day from this graph endpoint? current adapter is bit inflexible and we are not able to pull the data for missing days

Reply:
can you check if my refactor is correct?

Reply:
@g1nt0ki lgtm! yup for volumes you can check the query we use for internal api: https://github.com/galacticcouncil/HydraDX-api/blob/main/app/routes/defillama/v1/volume.mjs